packages/web/src/components/collection/CollectionCard.tsx

```diff
@@ -89,9 +91,11 @@ export const CollectionCard = forwardRef(
       playlist_owner_id,
       repost_count,
       save_count,
-      is_private,
+      is_private: isPrivate,
       access,
-      stream_conditions
+      stream_conditions,
+      is_scheduled_release: isScheduledRelease,
+      release_date: releaseDate
```

Comment:
feels inconsistent that we rename some but not others. I don't super care which we use

Reply:
yeah agree, my opinion is we should always use camelcase, but wasn't gonna touch other code unnecessarily